slices/sensible-utils.yaml

```diff
@@ -0,0 +1,27 @@
+package: sensible-utils
+
+essential:
+  - sensible-utils_copyright
+
+slices:
+  bins:
+    contents:
+      /usr/bin/select-editor:
+      /usr/bin/sensible-browser:
+      /usr/bin/sensible-editor:
+      /usr/bin/sensible-pager:
+      /usr/bin/sensible-terminal:
+      /usr/share/sensible-utils/bin/gettext:
+
+  mime:
+    contents:
+      /usr/lib/mime/packages/sensible-utils:
+
+  copyright:
+    contents:
+      /usr/share/doc/sensible-utils/copyright:
+
+  standard:
```

Comment:
yet another case where this time, it feels like `standard` is actually delivering "all".

I get the usefulness of the modularity, but given that `standard` and `core` are quite vague words, I'd refrain from using them unless they are actually delivering some standard functionality (like the `standard` slice in the `python3` slices`)

In this case (and others above) I'd suggest simply adding the `bins` and `mime` slices as dependencies (instead of the `standard` slice)